src/modules/capture/mod.rs

```diff
@@ -184,8 +184,8 @@ static BXT_CAP_STOP: Command = Command::new(
 );
 
 fn cap_stop(marker: MainThreadMarker) {
+    let mut state = STATE.borrow_mut(marker);
     unsafe {
-        let mut state = STATE.borrow_mut(marker);
```

Comment:
I am sometimes using unsafe blocks as regular blocks that happen to need unsafe, i.e. to also restrict the variable scope. In particular, here `state` is dropped at the end of the `unsafe {}` block, which is important, as right below `STATE` is accessed again, which will now cause a `RefCell` non-exclusive-mut-borrow panic.

Therefore, I'm afraid, this change is incorrect.